arbitrum-docs/bold/bold-gentle-introduction.md

```diff
@@ -0,0 +1,139 @@
+
+**What is the user flow for using the assertion staking pool contract?**
+
+- Anyone can deploy an assertion staking pool using [`AssertionStakingPoolCreator.sol`](https://github.com/OffchainLabs/bold/blob/main/contracts/src/assertionStakingPool/AssertionStakingPoolCreator.sol) as a means to crowdsource funds for staking on an assertion. To defend Arbitrum using a staking pool, an entity would first deploy this pool with the assertion that they believe is correct and wish to stake on to challenge an advesary's assertion. Then, anyone can verify that the claimed assertion is correct by running the inputs through their node's State Transition Function (STF). If other parties agree on the assertion being correct, then they can deposit their funds into the contract. When enough funds have been deposited, anyone can permissionlessly trigger the creation of the assertion on-chain to start the challenge. Finally, once the honest parties' assertion is confirmed by the dispute protocol, all involved entities will get their funds reimbursed and can withdraw.
+
+**Are there any incentives to run a BOLD validator to secure Arbitrum chains?**
+
+- Running a BOLD validator secures their respective Arbitrum chain and protects the assets on the chain from malicious actors - all you need is 1 honest party. Other than this critical piece, there are currently no financial incentives for parties to run a BOLD validator. Any future decisions or changes to this design can be proposed to and voted on by the Arbitrum DAO.
+
+**What type of hardware will be necessary to run a BOLD validator?**
+
+- The type of hardware required for running a BOLD validator is still being researched and finalized. The goal, however, is that regular consumer hardware (i.e. laptop) can effectively be used by an honest party to secure an Arbitrum chain using BOLD in the average case.
+
+**Is it possible that a high volume of disputes overwhelms the honest party?**
+
+- Yes - while BOLD is specifically designed to manage and resolve concurrent challenges between any number of parties, there is a chance that the high volume of challenges going on at the same time overwhelms the honest party. To mitigate this risk, the cost of opening a challenge will scale with the number of open challenges so as to make it unfeasible for a malicious party to perform a DoS attack on an Arbitrum chain, and well-resourced honest parties will be able to respond to this number of malicious claims.
```

Comment:
This is more a marketing issue, but as a user I would feel more secure without the first "Yes", and reading only the rest of the text.